File: 2.0/java/contacts/createContactListFromFile.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.ResourceId;

import java.io.File;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        File file = new File("/contacts.csv");
        ResourceId resourceId = client.contactListsApi().createFromCsv("My Contact List", file);
    }
}

File: 2.0/csharp/contacts/createContactListFromFile.cs
using System.IO;
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var path = "/contacts.csv";
        ResourceId listId = client.ContactListsApi.CreateFromCsv("My Contact List", Path.GetFullPath(path));
    }
}
